benchmark_thresholds.json

```diff
@@ -0,0 +1,3 @@
+{
+    "renatav/performance-tests": 15
```

Comment:
question: can we bring this threshold to be lower? Is 15 times slower before it errors out too much?

Reply:
That's 15%.  The default is 10%, this is just an example of how to overwrite the default value